pyface/tasks/action/listening_action.py

```diff
@@ -12,4 +12,4 @@
 Please use :class:`pyface.action.listening_action.ListeningAction` instead.
 """
 
-from pyface.action.listening_action import ListeningAction
+from pyface.action.listening_action import ListeningAction  # noqa: F401
```

Comment:
Should importing this module raise a `DeprecationWarning`?

Reply:
One more issue needed - to dig around and decide on if/when the module can be chucked out.

Reply:
opened - https://github.com/enthought/pyface/issues/917